### PR TITLE
Create utility to determine if a category is essential or not

### PR DESCRIPTION
**Pull request by ajanian**
There is a small commit to nav.html that moves the script referencing to head and formats one section.  Other than that everything is in the commit messages

### CODE REVIEW

File: templates/nav.html [hunk 1]
```diff
@@ -60,15 +58,15 @@
           <a class="nav-link text-light" href="/bydate.html?datafilename=raw">Chart/All</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link text-light" target="_blank" href="https://datastudio.google.com/s/n2u7zjB6voE">View in Data Studio</a>
+          <a class="nav-link text-light" href="/report_tree.html">Static Reports</a>
```
Comment (dsjoerg) [inline templates/nav.html]: Did you mean to change this line?
I think this was a recent change from @markroth8 ?

Comment (ajanian) [reply templates/nav.html]: no, I didn't.  that is a merge conflict :-(

Comment (dsjoerg) [reply templates/nav.html]: And more generally are there any changes to `nav.html` that are intentionally part of this PR?

Comment (ajanian) [reply templates/nav.html]: no.  no actual changes intended.

Comment (ajanian) [reply templates/nav.html]: looks like the only changes are formatting changes at this point

**Comment by dsjoerg** [comment]
OK since no changes to `nav.html` are intended, why don't you update this PR so that there are no changes to `nav.html` ?

**Comment by ajanian** [comment]
I don't know how to do that

File: templates/nav.html [hunk 2]
```diff
@@ -1,47 +1,45 @@
...
+      -moz-user-select: none;
+      -ms-user-select: none;
+      user-select: none;
+    }
+  </style>
+  <!-- Custom styles for this template -->
+  <link href="/css/visitdata.css" rel="stylesheet">
+  </body>
 
-    <title>Location Visit Data</title>
```
Comment (dsjoerg) [inline templates/nav.html]: for example this line was changed; this `title` tag was removed

**Pull request merged**